website/docs/FAQ.md

```diff
@@ -18,6 +18,32 @@ Currently FLAML does several things for imbalanced data.
 2. We use stratified sampling when doing holdout and kf.
 3. We make sure no class is empty in both training and holdout data.
 4. We allow users to pass `sample_weight` to `AutoML.fit()`.
+5. User can customize the weight of each class by setting the `custom_hp` or `fit_kwargs_by_estimator` arguments. For example, the following code sets the weight for pos vs. neg as 2:1 for the RandomForest estimator:
+
+```python
+from flaml import AutoML
+from pandas.tests.extension.conftest import as_frame
+from sklearn.datasets import load_iris
+
+X_train, y_train = load_iris(return_X_y=True, as_frame=as_frame)
+automl = AutoML()
+automl_settings = {
+    "time_budget": 2,
+    "task": "classification",
+    "log_file_name": "test/iris.log",
+    "estimator_list": ["rf"],
+    "max_iter": 0,
+}
+
+automl_settings["custom_hp"] = {
+    "rf": {
+        "class_weight": {
+            "domain": {0:2, 1:1}
```

Comment:
```suggestion
            "domain": {0: 2, 1: 1}
```

Reply:
Did you verify that the class_weight in rf is set to this dict? Could you add `print(automl.model)` in the end? And add this example to unit test.